README.md

```diff
@@ -23,7 +23,7 @@ options object.
   The maximum amount of time (seconds) the Reader will backoff for any single backoff
   event.
 * ```maxAttempts: 0``` <br/>
-  The number of times to a message can be requeued before it will be handed to the DISCARD handler and then automatically finished. 0 means that there is **no limit.** If no DISCARD handler is specified and `maxAttempts > 0`, then the message will be finished automatically when the number of attempts has been exhausted.
+  The number of times a given message will be attempted (given to MESSAGE handler) before it will be handed to the DISCARD handler and then automatically finished. 0 means that there is **no limit.** If no DISCARD handler is specified and `maxAttempts > 0`, then the message will be finished automatically when the number of attempts has been exhausted.
```

Comment:
I'm confused, I tested with [gist](https://gist.github.com/xavierchow/2223b6411301b7b29ec56ed86d09170e),
from the [output result](https://gist.github.com/xavierchow/2223b6411301b7b29ec56ed86d09170e#file-nsq-consumer-js-L25-L33), you can see the message handler gets executed 2 times and the `msg.attempts` is 2 as well, while the `maxAttempts` is set as 3.
I suppose the `msg.attempts` reaches 3 before given to the handler according the [nsqjs source code](https://github.com/dudleycarr/nsqjs/blob/26ea437f06ff4e881c905ff1b0e455fd6b2f57c7/src/reader.coffee#L150).
Is there anything I missed?